### PR TITLE
Initial helm chart for Harbor

### PR DESCRIPTION
**Pull request by paulczar**

### CODE REVIEW

**Comment by paulczar** [comment]
@jessehu

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,3 @@
-harbor
```
Comment (jessehu) [inline .gitignore]: does this ignore the directory contrib/helm/harbor ? I think harbor team ignores dir 'harbor' for some purpose. How about naming it 'contrib/helm-harbor'?

File: contrib/helm/harbor/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+- vmware
+- docker
+- registry
+home: https://github.com/vmware/harbor
+icon: https://github.com/vmware/harbor/blob/master/docs/img/harbor_logo.png
+sources:
+- https://github.com/vmware/harbor
+maintainers:
+- name: paulczar
+  email: username.taken@gmail.com
```
Comment (jessehu) [inline contrib/helm/harbor/Chart.yaml]: Is username.taken@gmail.com your personal email ? If yes, it is really great!

File: contrib/helm/harbor/README.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
+# VMWare Harbor
```
Comment (jessehu) [inline contrib/helm/harbor/README.md]: VMWare Harbor => Project Harbor by VMware

File: contrib/helm/harbor/README.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+[VMWare Harbor](http://vmware.github.io/harbor/) is an enterprise-class registry server that stores and distributes Docker images. Harbor extends the open source Docker Distribution by adding the functionalities usually required by an enterprise, such as security, identity and management. As an enterprise private registry, Harbor offers better performance and security. Having a registry closer to the build and run environment improves the image transfer efficiency. Harbor supports the setup of multiple registries and has images replicated between them. In addition, Harbor offers advanced security features, such as user management, access control and activity auditing.
```
Comment (jessehu) [inline contrib/helm/harbor/README.md]: VMWare Harbor => Project Harbor by VMware

File: contrib/helm/harbor/Chart.yaml [hunk 5]
```diff
@@ -0,0 +1,16 @@
+name: harbor
+version: 0.0.1
+appVersion: 1.3.0
+description: VMWare Enterprise Grade Docker Registry
+keywords:
+- vmware
+- docker
+- registry
```
Comment (jessehu) [inline contrib/helm/harbor/Chart.yaml]: please add the keyword 'harbor' as well.

File: .gitignore (hunk 1 above)
Comment (paulczar) [reply .gitignore]: in order to pass `helm lint` the directory the chart belongs in must be called `harbor`.  If the harbor team can provide the exact location of the `harbor` file they ignore I can make this ignore more specific.

File: contrib/helm/harbor/Chart.yaml (hunk 2 above)
Comment (paulczar) [reply contrib/helm/harbor/Chart.yaml]: yes

File: .gitignore (hunk 1 above)
Comment (jessehu) [reply .gitignore]: Got it. We can try adding '!/harbor' in contrib/helm/.gitignore or '!/contrib/helm/harbor' in .gitignore.

**Pull request merged**